tests/suites/test_suite_md.function

```diff
@@ -114,6 +128,8 @@ void md_info(int md_type, char *md_name, int md_size)
     (void) md_name;
 #endif
 
+    /* Note: PSA Crypto init not needed to info functions */
```

Comment:
Very minor:
```suggestion
    /* Note: PSA Crypto init not needed for info functions */
```

Reply:
I'll fix that in a follow-up in order to avoid another round of CI here.

Reply:
https://github.com/Mbed-TLS/mbedtls/pull/7271/commits/6ea8d3414fc835b5490ef048a419ac2c5454dc2a